### DISCUSSION TITLE
Manage callable functions with Rowy

### DISCUSSION THREAD

**Discussion by st-trade** [AntlerVC/firetable#772] [Idea / Feature Request]
Is there a way, or perhaps, some sort of workaround using rowy to [manage callable Cloud functions](https://firebase.google.com/docs/functions/callable)?

**Comment by harinij** [comment]
@st-trade in the action field you can connect to a callable function, but we don't have a way to manage it via the UI. Could you share a bit about the scenario under which you use them, as that will help us better understand the usecase.

**Comment by bofeiw** [comment]
Callable functions are not managed by Rowy. Here is how you can deploy a callable function and use it in the Rowy Action field: https://docs.rowy.io/field-types/action#callable
